app/assets/javascripts/streams.js

```diff
@@ -8,7 +8,10 @@ function timeAgoFormat() {
   });
 }
 
-$(document).ready(timeAgoFormat);
+$(document).ready(function() {
+  timeAgoFormat();
+  setInterval(timeAgoFormat, 60000); // update times every 60s
```

Comment:
this seems to be global, time ago is used in a couple of places, and it doesn't particularly hurt them actually being up-to-date... (is my thinking)